scripts/yaml_characters

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bin/rails runner
+# frozen_string_literal: true
+
+excluded_locales = []
+excluded_gem_paths = []
+OptionParser.new do |opts|
+  opts.on('--exclude-locale=LOCALE', 'Disregard characters from the given locale') do |locale|
+    excluded_locales << locale.to_sym
+  end
+
+  opts.on('--exclude-gem-path=GEM', 'Disregard characters loaded by the given gem') do |gem|
+    excluded_gem_paths << Gem.loaded_specs[gem].full_gem_path
+  end
+end.parse!(ARGV)
+
+def sanitize(string)
+  string.gsub(/<.+?>/, '').gsub(/%{.+?}/, '').gsub(/[\n\r\t]/, '')
+end
+
+def hash_values(hash)
+  hash.values.flat_map do |value|
+    case value
+    when Hash
+      hash_values(value)
+    when Array
+      value
+    else
+      [value]
+    end
+  end
+end
+
+I18n.load_path.reject! do |load_path|
+  excluded_gem_paths.any? { |gem_path| load_path.start_with?(gem_path) }
+end
+
+I18n.backend.eager_load!
+
+data = I18n.backend.translations.slice(*I18n.available_locales - excluded_locales)
```

Comment:
IMO we should do something similar to this in `spec/i18n_spec.rb`. Interestingly `i18n-tasks` doesn't load all string data eagerly like this, so we miss some strings in those specs. Having a more complete set of strings to test against could help prevent issues like [one we encountered a few weeks back](https://gsa-tts.slack.com/archives/C0NGESUN5/p1714574889100899).

Reply:
makes sense to me to add that there